angular.json

```diff
@@ -108,6 +108,7 @@
   },
   "defaultProject": "cvs-app-vtm",
   "cli": {
+    "analytics": false,
```

Comment:
What is this for?

Reply:
Remember when we re-installed the packages using node v14. Since then, this has been asking me every time if Google can collect analytics. This turns that off for good.